lib/jobs/jira-sync.ts

```
@@ -0,0 +1,179 @@
+          syncStatus: 'success',
+          syncError: null,
+          jiraUpdatedAt: new Date(jiraIssue.fields.updated),
+          fieldsHash,
+          updatedAt: new Date()
+        })
+        .where(eq(jiraTicketSync.id, existingSync[0].JiraTicketSync.id));
+    }
+  }
+
+  private static generateFieldsHash(ticketData: any): string {
+    const crypto = require('crypto');
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)